pkgs/applications/window-managers/dwl/default.nix

```diff
@@ -40,18 +47,16 @@ stdenv.mkDerivation rec {
     wayland
     wayland-protocols
     wlroots
-  ] ++ lib.optionals enable-xwayland [
+  ] ++ lib.optionals enableXWayland [
     libX11
+    xcbutilwm
     xwayland
   ];
 
-  # Allow users to set their own list of patches
-  patches = totalPatches;
+  outputs = [ "out" "man" ];
 
-  # Last line of config.mk enables XWayland
-  prePatch = lib.optionalString enable-xwayland ''
-    sed -i -e '$ s|^#||' config.mk
-  '';
+  # Allow users to set their own list of patches
+  patches = extraPatches ++ [];
```

Comment:
Why are we keeping this? We have `overrideAttrs`?

Reply:
Better: I will remove it.